src/core/service/notifications/admin/handler.go

```diff
@@ -108,10 +107,7 @@ func (h *Handler) HandleAdminJob() {
 		return
 	}
 
-	// For scan all job
-	if h.jobName == job.ImageScanAllJob && h.checkIn != "" {
-		go scan.HandleCheckIn(orm.NewContext(context.TODO(), o.NewOrm()), h.checkIn)
-	} else if h.jobName == job.ImageGC && h.status == models.JobFinished {
+	if h.jobName == job.ImageGC && h.status == models.JobFinished {
```

Comment:
we decide to deprecate the admin job in 2.2, will you address another PR to remove the admin job dependency?

Reply:
I think`quota.RefreshForProjects(orm.NewContext(context.TODO(), o.NewOrm()))` needs to be changed in the refractor of gc.

And remove the admin job in another PR.